src/strategies/sushiswap/strategy-sushi-eth-sushi-lp.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+import "../strategy-sushi-farm-base.sol";
+
+contract StrategySushiEthSushiLp is StrategySushiFarmBase {
+    // Token/ETH pool id in MasterChef contract
+    uint256 public sushi_eth_poolId = 12;
+    // Token addresses
+    address public sushi_eth_sushi_lp = 0x795065dCc9f64b5614C407a6EFDC400DA6221FB0;
+
+    constructor(
+        address _governance,
+        address _strategist,
+        address _controller,
+        address _timelock
+    )
+        public
+        StrategySushiFarmBase(
+            sushi,
```

Comment:
There is a bit of an inefficiency here. This is because the jar will sell the sushi that it has harvested for eth, then use half of the eth to buy sushi again. Instead , we could just sell half of the sushi for eth and use other half to autocompound  .This is due to the way that the underlying strategy is implemented(it sells sushi for token1 (which in this case is sushi again )

Reply:
This would end up increasing gas costs for the jar and making it less profitable. This could be solved by changing the underlying strategy contract (would this require another audit) to use the mechanism above.